packages/downloads/src/portal/build-existing-exports-portal-query.ts

```diff
@@ -0,0 +1,149 @@
+import { SearchQueryBuilder } from "@esri/arcgis-rest-portal";
+
+const WGS84_WKID = '4326';
+
+export const PORTAL_EXPORT_TYPES = {
+  csv: {
+    name: 'CSV',
+    itemType: 'CSV Collection',
```

Comment:
I think there are types `CSV` and `CSV Collection`, as well as `KML` and `KML Collection`.  If I recall correctly, Portal assigns these types dependent on whether the export is a single layer or multi-layer.  So your search query will likely need to take this into account.